cpp/models/sde_sir/README.md

```diff
@@ -0,0 +1,21 @@
+
+# ODE SIR compartment model
+
+This model is a very simple ODE model with only three compartments and few parameters, mostly for demonstration of the MEmilio framework:
+- Susceptible, may become infected at any time
+- Infected, will be recovered after some time
+- Recovered, recovered from infectious process (dead or recovered)
+
+We assume simulations over short periods of time, so that the population size can be considered constant and birth as well as (natural) mortality rates can be ignored. 
+
+Below is an overview of the model architecture and its compartments.
+
+![SIR_model](https://github.com/SciCompMod/memilio/assets/69154294/01c9a2ae-2f5c-4bad-b7f0-34de651f2c73)
```

Comment:
This picture should be removed/replaced. We need one without the ODE parameters on the arrows. @HenrZu Can you help out?